uaclient/cli/tests/test_cli_fix.py

```diff
@@ -145,6 +145,7 @@ def test_cve_header(self):
         cve = FixPlanResult(
             title="CVE-2020-1472",
             description="Samba vulnerability",
+            current_status="system-not-affected",
```

Comment:
I think this would be "not-affected" now (?)

Reply:
This one might've slipped through, it should be not-affected